src/fides/lib/oauth/system_manager.py

```diff
@@ -0,0 +1,5 @@
+from fides.api.ops.api.v1.scope_registry import SYSTEM_DELETE, SYSTEM_UPDATE
+
+# System managers are separate from roles, because you are just granted these
+# permissions to specific system(s) not to all resources of a given type
+SYSTEM_MANAGER_SCOPES = [SYSTEM_UPDATE, SYSTEM_DELETE]
```

Comment:
Starting with these scopes for system managers and we can continue to expand as extend what they can do when they've been assigned to a system.